app/src/main/java/com/hammer/talkbbokki/presentation/main/MainScreen.kt

```diff
@@ -159,15 +159,15 @@ fun MainHeader(
 @Composable
 fun LevelItem(
     level: CategoryLevel,
-    onClickLevel: (String) -> Unit,
+    onClickLevel: (String, String) -> Unit,
     showDialog: () -> Unit
 ) {
     Box(modifier = Modifier.aspectRatio(1f)) {
         Card(
             modifier = Modifier.fillMaxSize(),
             shape = RoundedCornerShape(8.dp),
             onClick = {
-                if (level.isActive) onClickLevel(level.id) else showDialog()
+                if (level.isActive) onClickLevel(level.id, (level.title).replace("\n","**")) else showDialog()
```

Comment:
🚫 **[ktlint]** <max-line-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Exceeded max line length (100) (cannot be auto-corrected)